cloud/services/loadbalancers.go

```diff
@@ -0,0 +1,141 @@
+package services
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"slices"
+	"strconv"
+
+	"github.com/go-logr/logr"
+	"github.com/linode/cluster-api-provider-linode/cloud/scope"
+	"github.com/linode/cluster-api-provider-linode/util"
+	"github.com/linode/linodego"
+)
+
+var (
+	defaultLBPort = 6443
```

Comment:
```suggestion
	defaultNBPort = 6443
```

Reply:
thoughts on NB everywhere vs. going back and forth? Maybe LB everywhere is right since NodeBalancer is the implementation?

Reply:
the consistency I was thinking of here is if a setting could be applied to any load balancer type supported by this provider then use `LB` and if it was specific to setting up nodebalancers themselves and wouldn't apply to other future loadbalancer types then use `NB` happy to standardize on just NB if we want to kick future use cases down the road